docs/contributing/backend/overview.mdx

```diff
@@ -60,12 +60,10 @@ root@b9d811e1e3e8:/home/src# python3 -m unittest discover -s mage_ai.tests --fai
 ### API
 
 - [Overview](/api-reference/overview)
-- [Routes](/api-reference/routes)
```

Comment:
did you mean to delete this?

Reply:
I couldn't find any docs for routes... so I assumed it was a artifact or outdated page